### ISSUE TITLE
The documentation for Invoke-Fzf is not up-to-date

### ISSUE THREAD

**Issue by pluma9** [kelleyma49/PSFzf#77]
The `Invoke-Fzf.md` file is not up-to-date. For example, it claims that there's a parameter called `InlineInfo`, but when I run `Invoke-Fzf -InlineInfo`, I got this error:

**Invoke-Fzf: A parameter cannot be found that matches parameter name 'InlineInfo'.**

**Comment by kelleyma49** [comment]
Good catch. `fzf` is constantly changing, which means I tend to fall before when it comes to options and documentation. I'll try to fix this when I have a period of uninterrupted time. :)

**Comment by harriott** [comment]
Also, under DESCRIPTION, it seems to have a line taken from the `fasdr` repository...
